targets/ipq40xx-generic

```diff
@@ -73,6 +73,15 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
+device('linksys-mr8300-dallas', 'linksys_mr8300', {
+	packages = { -- exclude until Gluon supports third radio
+		'-ath10k-firmware-qca9888-ct',
+		'-ath10k-board-qca9888',
+	},
+	factory = false,
+	broken = true, -- 3rd WiFi radio not supported (5GHz DFS channels 100+)
```

Comment:
I think this comment should be extended further:
- It would be nice to also state explicitly that the 2rd radio only supports the lower channels
- Also mention that outdoor mode with autochannel will be broken because of that